web/ui/navbar/index.svelte

```diff
@@ -49,28 +51,45 @@
 
 <nav class="navbar">
   <div class="navbar_main" aria-orientation="horizontal" role="menu">
-    {#if $isRefreshing}
+    <div
+      class={`navbar_back-btn ${$side === 'first' ? 'navbar_btn-hidden' : ''}`}
+    >
       <NavbarItem
-        name={$t.refresh}
-        current={$router.route === 'refresh'}
-        hotkey="r"
-        href={getURL('refresh')}
-        small
-      >
-        <NavbarProgress value={$refreshProgress} />
-      </NavbarItem>
-    {:else}
-      <NavbarItem
-        name={$t.refresh}
-        current={$router.route === 'refresh'}
-        hotkey="r"
-        icon={mdiRefresh}
+        name={$t.back}
+        current={true}
+        hotkey="Esc"
+        href={backRoutes[$router.route]}
```

Comment:
To get back URL here you can do something like: 

```js
getPagePath(urlRouter, backRoute)
```

backRoute will be from the core (which know nothing about URLs) and here we will combine route with web's URL router which has logic to convert abstract routes to URL.